frontend/src/Components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import "./SideBar.css";
+
+const Sidebar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleSidebar = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <>
+      <button className="menu-btn" onClick={toggleSidebar}>
+        ☰
```

Comment:
this  button (as well as the X button) should probably have `aria-label`s since screen readers would otherwise probably announce the unicode character name which isn't super helpful